vsts/node-release.yaml

```diff
@@ -0,0 +1,185 @@
+      git push --tags -u origin $($releaseBranch)
+      git checkout master
+      git branch -d $releaseBranch
+      
+    displayName: Create Release Branch
+    enabled: true
+
+  - powershell: |
+       runas.exe /savecred /user:administrator
+       npm install --global node-gyp
+       npm install --global lerna
+    displayName: 'Install Dependencies'
+    enabled: false
+
+  - script: |
+       call lerna bootstrap
+       call lerna run build
+    displayName: 'Bootstrap & Build'
+    enabled: false
+
+  - script: 'call lerna run ci'
+    displayName: 'Unit & Integration Tests'
+    env:
+      IOTHUB_CONNECTION_STRING: $(WINDOWS-IOTHUB-CONNECTION-STRING)
+    enabled: false
+
+  - script: 'lerna run e2e'
+    displayName: 'E2E Tests'
+    env:
+      IOTHUB_CA_ROOT_CERT: $(IOTHUB-CA-ROOT-CERT)
+      IOTHUB_CA_ROOT_CERT_KEY: $(IOTHUB-CA-ROOT-CERT-KEY)
+      IOT_PROVISIONING_DEVICE_ENDPOINT: $(IOT-PROVISIONING-DEVICE-ENDPOINT)
+      IOTHUB_CONNECTION_STRING: $(WINDOWS-IOTHUB-CONNECTION-STRING)
+      STORAGE_CONNECTION_STRING: $(WINDOWS-STORAGE-CONNECTION-STRING)
+      IOT_PROVISIONING_DEVICE_IDSCOPE: $(WINDOWS-IOT-PROVISIONING-DEVICE-IDSCOPE)
+      IOT_PROVISIONING_ROOT_CERT: $(WINDOWS-IOT-PROVISIONING-ROOT-CERT)
+      IOT_PROVISIONING_ROOT_CERT_KEY: $(WINDOWS-IOT-PROVISIONING-ROOT-CERT-KEY)
+      IOT_PROVISIONING_SERVICE_CONNECTION_STRING: $(WINDOWS-IOT-PROVISIONING-SERVICE-CONNECTION-STRING)
+      DPS_CONN_STRING_INVALID_CERT: $(DPS-CONN-STRING-INVALID-CERT)
+      DPS_GLOBAL_DEVICE_ENDPOINT_INVALID_CERT: $(DPS-GLOBAL-DEVICE-ENDPOINT-INVALID-CERT)
+      IOTHUB_CONN_STRING_INVALID_CERT: $(IOTHUB-CONN-STRING-INVALID-CERT)
+      IOTHUB_DEVICE_CONN_STRING_INVALID_CERT: $(IOTHUB-DEVICE-CONN-STRING-INVALID-CERT)
+    enabled: false
+
+  - task: PublishTestResults@2
+    displayName: 'Publish Test Results | Mocha'
+    inputs:
+      testResultsFiles: 'e2etests/test-results.*.xml'
+      mergeTestResults: true
+      testRunTitle: 'E2E Tests - Windows'
+    condition: succeededOrFailed()
+    enabled: false
+
+  - powershell: |
+      $folders = @(
+        'common/core',
+        'common/transport/amqp',
+        'common/transport/http',
+        'common/transport/mqtt',
+        'device/core',
+        'device/transport/amqp',
+        'device/transport/http',
+        'device/transport/mqtt',
+        'device/node-red',
+        'service',
+        'provisioning/service',
+        'provisioning/device',
+        'provisioning/transport/amqp',
+        'provisioning/transport/mqtt',
+        'provisioning/transport/http',
+        'security/x509',
+        'security/tpm',
+        'security/symmetric')
+
+      $repoParent = $(Get-Item "$(Agent.BuildDirectory)").Parent
+
+      $artifacts = '$(Agent.RootDirectory)\.tar'  # artifacts folder path 
+
+      if ($(Test-Path $artifacts -PathType Container) -eq $true) {
+          Remove-Item -Path $artifacts -Recurse  # delete old artifacts folder
+      }
```

Comment:
why should we do that if its' a new build? is this folder shared from one build to the next?